Sources/Collection.swift

```diff
@@ -27,7 +27,7 @@ public final class Collection {
         mongoc_collection_destroy(self.pointer)
     }
 
-    public func insert(document: BSON.Document, flags: InsertFlags = .None) throws {
+    public func insert(document document: BSON.Document, flags: InsertFlags = .None) throws {
```

Comment:
I'm waiting for the ci at BinaryJSON to pass (I just configured it for 03-24) before merging & tagging a release, but the version is going to be `1.5`